src/System.IO.FileSystem/tests/DirectoryInfo/CreateSubdirectory.cs

```diff
@@ -140,14 +140,27 @@ public void AllowedSymbols()
 
         [Theory,
             MemberData(nameof(ControlWhiteSpace))]
-        [PlatformSpecific(TestPlatforms.Windows)]  // Control whitespace in path throws ArgumentException
-        public void WindowsControlWhiteSpace(string component)
+        [PlatformSpecific(TestPlatforms.Windows)]
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.NetFramework)]
+        public void WindowsControlWhiteSpace_Desktop(string component)
         {
             // CreateSubdirectory will throw when passed a path with control whitespace e.g. "\t"
             string path = IOServices.RemoveTrailingSlash(GetTestFileName());
             Assert.Throws<ArgumentException>(() => new DirectoryInfo(TestDirectory).CreateSubdirectory(component));
         }
 
+        [ActiveIssue(27269)]
+        [Theory,
+            MemberData(nameof(ControlWhiteSpace))]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
+        public void WindowsControlWhiteSpace_Core(string component)
+        {
+            // CreateSubdirectory will throw when passed a path with control whitespace e.g. "\t"
+            string path = IOServices.RemoveTrailingSlash(GetTestFileName());
```

Comment:
I'm wondering why we have this path statement here in the existing test you copied when it's not going to throw exception? the input argument is ```component``` which is being used in the next statement not to compute path. 
Am I missing something?

Reply:
I just missed the fact that it wasn't used as I plowed through these. Good catch.